ros2service/ros2service/verb/call.py

```diff
@@ -77,23 +79,17 @@ def main(self, *, args):
             args.service_type, args.service_name, values, period, default_profile)
 
 
-def requester(service_type, service_name, values, period, qos_profile):
-    # TODO(wjwwood) this logic should come from a rosidl related package
+def requester(service_type: str, service_name: str, values, period: Optional[float],
+              qos_profile: QoSProfile) -> None:
     try:
         parts = service_type.split('/')
         if len(parts) == 2:
             parts = [parts[0], 'srv', parts[1]]
```

Comment:
I think we can remove this logic. It's not needed to get `package_name` and `srv_name` for the node name below (first and last components, respectively).

Reply:
I think I removed the logic you meant.

Reply:
Yep, thank you!